include/picongpu/plugins/openPMD/openPMDWriter.hpp

```diff
@@ -808,6 +808,7 @@ Please pick either of the following:
                     }
                 }
 
+#if(PMACC_CUDA_ENABLED == 1 || ALPAKA_ACC_GPU_HIP_ENABLED == 1)
```

Comment:
I am not sure if skipping this on non-GPU builds is safe to do. Can someone verify?

Reply:
Right now `dc.releaseData()` does nothing, it's literally empty implementation. I think this call can be just removed as well.

Reply:
Ok, I've removed those lines in the latest commit

Reply:
keeping it inside of the guard is correct. s @sbastrakov said it is currently an empty implementation but should keep in the code to simplify later refactoring where this function gets some functionality.

Reply:
Ohh Sry I have not read the full diff. 
The implementation is fine: For non-GPU devices the code must be skipped because we do not have a mallocMC buffer.